lacework/README.md

```diff
@@ -0,0 +1,39 @@
+# Agent Check: Lacework
```

Comment:
If we intend to store the integrations content here, let's be consistent with the current lacework integration documentation already in the doc: https://docs.datadoghq.com/integrations/lacework/

Source file: https://github.com/DataDog/documentation/blob/master/content/en/integrations/lacework.md